components/theme/Button.tsx

```diff
@@ -24,7 +39,9 @@ export const Button: React.FC<ButtonProps> = ({
   onClick = noop,
   children,
   size,
-  outline
+  outline,
+  testId,
```

Comment:
I don't think we need to create a prop for testId. It should be treated like the other html attributes(...other)